workshops/personal_website/README.md

```diff
@@ -200,7 +219,7 @@ Go ahead and add this into your `index.html` now. I put my picture before my hea
 
 Hurray! You've created a basic HTML page! Let's save our progress with git!
 
-In Cloud9, press `Alt + t` on the keyboard. This will open the terminal, which allows us to run special types of commands in Cloud9. In this case, we're going to save the work we've done so far, using git. Type in the following commands:
+In Cloud9, press `Alt + t` (`option + t` on mac) on the keyboard. This will open the terminal, which allows us to run special types of commands in Cloud9. In this case, we're going to save the work we've done so far, using git. Type in the following commands:
```

Comment:
They use "Mac" on their website when referring to keyboards: https://support.apple.com/en-us/HT201236